free-1.md

```diff
@@ -1,54 +1,76 @@
 # Free project 1
 
 ## The user and a language
-This section describes who the project would serve and why a language might be a
-good way to meet their needs.
+The first language I would like to consider is one for music composers. It would help them organize ideas about compositions into structures they create, simulating a top-down approach to composing music.
 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My idea satisfies the need of organization when composing music. We will be helping composers of any skill level organize their compositions. The main reason why I'd like to build a language for organizing composition structures is because a current person's experiences in composing is done on paper or in a language not built for organizing composition structures. There are a lot (like, really, a lot) of langauges that focus on creating music and aiding musicians. But I haven't come across one solely dedicated to organization yet. I hope that the language I am thinking of helps people efficiently organize their thoughts in compositions and frees them to think of other aspects of composing music such as harmony, rhythm, and textures. From here, the hope is that the less time spent thinking about organization leads to more time/efficiency on the other aspects.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+One reason why I see a DSL as a potential solution is because current music software includes a lot of other things. Many programming languages, libraries, and APIs that implement ways to create music focus on making it possible to make cool sounds and patterns. But instead of having that affect one's compositions (following the Sapir-Whorf hypothesis), having a programming langauge specific to enhancing one's compositional creativity would decrease the impact of languages on musician's compositions. A DSL can be used as a preliminary tool or a tool in conjunction with other music software. As stated above, the end goal is to let users easily think of problems other than the structure of a composition that arise when composing.
```

Comment:
What is the Sapir-Whorf hypothesis? I'm having a hard time following the exact purpose of your language and how it differs from other tools, and why that difference is significant, important, and useful.